pandas/tseries/offsets.py

```diff
@@ -603,61 +641,76 @@ def next_bday(self):
         else:
             return BusinessDay(n=nb_offset)
 
-    @cache_readonly
-    def _get_daytime_flag(self):
-        if self.start == self.end:
-            raise ValueError('start and end must not be the same')
-        elif self.start < self.end:
-            return True
-        else:
-            return False
+    def _get_daytime_flag(self, start, end):
```

Comment:
can you add a doc-string

Reply:
I removed this function as it's only use once and I think its clearer to just use `start < end`.